### ISSUE TITLE
Quick view doesn't show the popup with actions (e.g. logs) for users

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit-gui#215]
When a user selects a workflow task in the graph and tree views, the user sees a popup with more information (submit times) and some actions like visualize logs.

This is not available in the quick view.

Perhaps we could load the information for that popup on the background, asynchronously, once any experiment view is loaded. This way, if the user loaded any view then we could also display the same popup (i.e. it would be more consistent, as clicking on a task/job of a workflow in a View would always behave the same way, independently of the View).

At the moment I have to switch to another view if I am looking at quick view but need to look a the logs of the task executed by some testing suite experiment, which is a bit annoying :slightly_frowning_face:

**Comment by LuiggiTenorioK** [comment]
I understand that the quick view can be misleading :face_with_spiral_eyes:  

The idea of that view is just to give a quick minimal summary of what is going on with the experiment after all. So, I'm not sure if extending the amount of information in that view will be the best solution since upgrading it will feel more like a duplicate of the tree view.

One possible alternative is to disable the hover and pointer change when the cursor is on an item which makes the user feel that something is going to happen if clicked. WDYT?

**Comment by LuiggiTenorioK** [comment]
@kinow the misleading hovering has been removed on the latest version of the GUI (PR #232). You can see it at https://earth.bsc.es/autosubmitapp/

I'm not sure if that change was enough to close this issue. WDYT?

**Comment by kinow** [comment]
I think it'd be better if we had the popup everywhere, but if that's not possible, I think disabling the click in the quick view works for now.

Not sure if others may want the popup everywhere too -- if yes, maybe this issue should be left open to gather feedback, otherwise, I think this can be closed 👍 up to you.

Thanks Luiggi!

**Comment by LuiggiTenorioK** [comment]
Once https://github.com/BSC-ES/autosubmit/issues/2211 is done, it will be more feasible to do a popup without overloading the API too much.
